<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Построение эллипсов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="canvas">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="canonButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>79</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Каноническое уравнение</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="paramButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>99</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Параметрическое уравнение</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="brezButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>19</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Брезенхем</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="middleButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>39</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Средняя точка</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="libraryButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>59</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Библиотечная функция</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>180</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="CoordLabelR">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>200</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Координаты центра</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="RXspinBox">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>200</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="maximum">
     <number>560</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="RYspinBox">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>200</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="minimum">
     <number>255</number>
    </property>
    <property name="maximum">
     <number>560</number>
    </property>
   </widget>
   <widget class="QLabel" name="RadiusLabelR">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>230</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Радиус</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="RRspinBox">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>230</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QPushButton" name="lineColorButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>490</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Выбрать цвет линии</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bgColorButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>450</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Выбрать цвет фона</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="bground_color">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>460</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QGraphicsView" name="line_color">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>500</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="clean_all">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>530</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>164</red>
          <green>6</green>
          <blue>45</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>164</red>
          <green>6</green>
          <blue>45</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>148</red>
          <green>148</green>
          <blue>148</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Очистить холст</string>
    </property>
   </widget>
   <widget class="QPushButton" name="roundButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>260</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Отобразить окружность</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="EYspinBox">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>340</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="maximum">
     <number>560</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="CoordLabelE">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>340</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Координаты центра</string>
    </property>
   </widget>
   <widget class="QLabel" name="HalfsLabelE">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>370</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Полуоси</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="EXspinBox">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>340</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="maximum">
     <number>560</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="EHXspinBox">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>370</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="EHYspinBox">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>370</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPushButton" name="ellipseButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>400</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Отобразить эллипс</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>440</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>310</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>120</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="RadiusLabelR_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>140</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Количество</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="CountBox">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>140</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="RadiusLabelR_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>160</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Шаг</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="StepBox">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>160</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>15</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
